app/controllers/projects_controller.rb

```diff
@@ -18,18 +18,22 @@ def index
 
   # GET /projects/1
   def show
-    @project = Project.includes(:project_status, members: [:projects_users]).find(params[:id])
+    @project = Project.includes(:project_status, :categories, members: [:projects_users]).find(params[:id])
     @serializer_options[:include] = [:members]
     @serializer_options[:params] = { project_id: params[:id] }
     render json: ProjectSerializer.new(@project, @serializer_options).serialized_json
   end
 
   # POST /projects
   def create
+    @categories = Category.find(params[:project][:categories])
+
     @project = Project.new(project_params)
+
     @project.projects_users.build(admin: true, user_id: auth_user.id)
 
     if @project.save
+      @project.categories << @categories
       render json: ProjectSerializer.new(@project).serialized_json
     else
       render json: ErrorSerializer.new(@project.errors, Rack::Utils.status_code(:unprocessable_entity)).serialized_json, status: :unprocessable_entity
```

Comment:
```Rack::Utils``` va messo a inizio pagina come import, in modo da poter chiamare direttamente ```status_code(...)``` nella pagina